docs/user/communication.rst

```diff
@@ -182,6 +193,27 @@ Reference Lecture Attachments
 Users can refer to lectures of the current course, via the dropdown menu ``Lecture`` available on the posting markdown
 editor (see image above). Here, lecture attachments can be found in a nested structure.
 
+Reference Lecture Attachment Units
+""""""""""""""""""""""""""""""""""
+
+Users can refer to lecture attachment units of the current course, via the dropdown menu ``Lecture`` available on the posting markdown
+editor (see image below). Here, lecture attachment units can be found when users hover over the specific lecture.
+
+Reference Lecture Unit Slides
+"""""""""""""""""""""""""""""
+
+Users can refer to lecture unit slides of the current course, via the dropdown menu ``Lecture``. Here, slides can be found when users
+hover over a specific unit (see image below).
+
+|slide-reference-menu|
+
+After the user references a single slide they can see as image included in the message. Additionally they can preview the slide
+in order to easily read the content by clicking the image.
```

Comment:
```suggestion
After the user references a single slide they can see an image included in the message. Additionally, they can preview the slide
in order to easily read the content by clicking the image.
```

Reply:
Maybe it should be `they can see it as an image`?